packages/expo-ui/components/Progress/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { requireNativeView } from 'expo';
+import { StyleProp, ViewStyle } from 'react-native';
+
+export type ProgressProps = {
+  /**
+   * Custom styles for the progress component.
+   */
+  style?: StyleProp<ViewStyle>;
+  /**
+   * The current progress value of the slider. This is a number between 0 and 1.
+   */
+  progress?: number | null;
+  /**
+   * Progress color.
+   */
+  color?: string;
+
+  /**
+   * Track color.
+   *
+   * @platform android
+   */
+  trackColor?: string;
+};
+
+type NativeProgressProps = ProgressProps & {
+  variant: 'linear' | 'circular';
+};
+
+const NativeProgressView: React.ComponentType<NativeProgressProps> = requireNativeView(
+  'ExpoUI',
+  'ProgressView'
+);
+
+export function CircularProgress(props: ProgressProps) {
+  return <NativeProgressView {...props} variant="circular" />;
```

Comment:
If you're experiencing some flicker on Android due to measure delays (and from experience that seems to happen), you can provide a default size here for the first frame (from material-ui guidelines).

Reply:
I haven't noticed a flicker, maybe we can add that later if we notice it is a problem (or maybe a better fix would be to see if there's a way to avoid the flicker altogether).